dev-db/kdb/patches/kdb-3.2.0.patchset

```diff
@@ -161,5 +160,29 @@ index c035b16..14f61ed 100644
  include(KDEFrameworkCompilerSettings NO_POLICY_SCOPE)
  
 -- 
-2.42.1
+2.50.1
+
+
+From 3a0b30ead00782edc9ceec41be43b070a7bd1140 Mon Sep 17 00:00:00 2001
+From: Luc Schrijvers <begasus@gmail.com>
+Date: Sat, 6 Sep 2025 07:48:20 +0200
+Subject: Fix build for cmake > 4
+
+
+diff --git a/cmake/modules/SetKDbCMakePolicies.cmake b/cmake/modules/SetKDbCMakePolicies.cmake
+index bc28d8a..bd400c2 100644
+--- a/cmake/modules/SetKDbCMakePolicies.cmake
++++ b/cmake/modules/SetKDbCMakePolicies.cmake
+@@ -8,9 +8,6 @@ cmake_policy(SET CMP0048 NEW) # for PROJECT_VERSION
+ cmake_policy(SET CMP0053 NEW) # TODO remove, temporary fix for a bug in Qt 5.8's Qt5ModuleLocation.cmake
+                               # "Simplify variable reference and escape sequence evaluation"
+ 
+-if(POLICY CMP0059) # Don’t treat DEFINITIONS as a built-in directory property.
+-    cmake_policy(SET CMP0059 OLD)
```

Comment:
`DEFINITIONS` doesn't seem to be used in cmake files, so LGTM. Although upstream did change it to `NEW`: https://github.com/KDE/kdb/commit/e2ccc568120a3700d28e12345bcf17e2faf96dd2, but it is unclear to me why this was even added in the first place, presumably in https://github.com/KDE/kdb/commit/76e3dc897ab1039e2c00beba3f169ecd613be863.